play.pokemonshowdown.com/testclient-beta.html

```diff
@@ -102,7 +102,7 @@ <h3><button class="closebutton" tabindex="-1" aria-label="Close"><i class="fa fa
 	<script nomodule src="/js/lib/ps-polyfill.js"></script>
 	<script src="../config/testclient-key.js"></script>
 	<script src="js/client-core.js" onerror="alert('You must build the client with `node build` before using testclient.html')"></script>
-
+	<script src="data/text-afd.js"></script>
```

Comment:
This should be moved next to `data/text.js`. And also added to the same place in `preactalpha.template.html`.